src/display/editor/drawers/freedraw.js

```diff
@@ -734,9 +411,53 @@ class FreeOutliner {
       this.#isLTR
     );
   }
+
+  _getOutlineStart(outline, pos) {
```

Comment:
Can we make this method actually private instead?
```suggestion
  #getOutlineStart(outline, pos) {
```